pytorch_forecasting/metrics/distributions_pkg.py

```diff
@@ -0,0 +1,143 @@
+"""
+Package containers for all distribution-forecast metrics.
+"""
+
+import torch
+
+from pytorch_forecasting.data import TorchNormalizer
+from pytorch_forecasting.metrics.base._base_object import _BasePtMetric
+
+
+class NormalDistributionLoss_pkg(_BasePtMetric):
```

Comment:
can you kindly change the pkg files to private for now? This will make it easier to later move them around without breaking user code.